src/theme/components/template.ts

```diff
@@ -24,27 +29,64 @@ const Template = defineStyleConfig({
   variants: {
     left: {
       gridTemplateAreas: {
-        base: `"breakout" "top" "sidebar" "main" "bottom"`,
-        md: `"breakout breakout" "top top" "sidebar main" "bottom bottom"`,
-      },
-      gridTemplateColumns: {
-        base: "100%",
-        md: "repeat(2, minmax(100px, 1fr))",
-        lg: "minmax(100px, 1fr) minmax(200px, 2fr)",
-        xl: "minmax(100px, 1fr) minmax(300px, 3fr)",
+        base: `
+          "breakout"
+          "top"
+          "sidebar"
+          "main"
+          "bottom"
+        `,
+        sm: `
+          "breakout breakout breakout breakout breakout breakout breakout breakout breakout breakout breakout breakout" 
```

Comment:
Is there a different way to write this? It _looks_ brittle, and I imagine and I need to update this and lose count, I can introduce a layout issue.

Reply:
Yes, when the wrong number of names are included, the layout blows up and it can be hard to figure out where the issue is coming from. There are other methods that will accomplish the same thing (i.e. named grid lines and spans). That other method would still require accuracy to calculate the correct starting point and span values, but maybe the whole page won't get out of whack if an incorrect value is used. I will do some testing with that method and see what happens.